indico/modules/core/controllers.py

```diff
@@ -30,7 +30,7 @@
 from indico.modules.cephalopod import cephalopod_settings
 from indico.modules.core.forms import ReportErrorForm, SettingsForm
 from indico.modules.core.settings import core_settings, social_settings
-from indico.modules.core.views import WPContact, WPSettings
+from indico.modules.core.views import WPContact, WPSettings, WPSemanticExamples
```

Comment:
this makes Travis builds fail

Reply:
It's `isort` complaining because of `WPSemanticExamples` which is gonna be removed anyway.

Reply:
Yeah, yeah I know :P just saying